apps/site-projects/src/components/modules/Projects/Project/EditorNotification/StyledEditorNotification.js

```diff
@@ -0,0 +1,62 @@
+
+import styled from "styled-components";
+
+
+export const Wrapper = styled.div`
+   font-family: "Abel";
+  align-content: center;
+  text-align: center;
+  display: grid;
+  /* flex-wrap: wrap ; */
+  row-gap: 1rem;
+  margin: auto;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  h3 {
+    align-content: center;
+    text-align: center;
+    margin: auto;
+    font-size: 2rem;
+    }
+ p {
+    align-content: center;
+    text-align: center;
+    margin: auto;
+    /* flex: 4; */
```

Comment:
can we delete this and the one on line 10?

Reply:
Done!